src/overlays/actors/ovl_En_Ishi/z_en_ishi.c

```diff
@@ -139,215 +140,235 @@ static InitChainEntry sInitChain[][5] = {
     },
 };
 
-static u16 D_8095F7AC[] = { NA_SE_PL_PULL_UP_ROCK, NA_SE_PL_PULL_UP_BIGROCK };
+static u16 sIshiPullRockSfx[] = { NA_SE_PL_PULL_UP_ROCK, NA_SE_PL_PULL_UP_BIGROCK };
 
-static EnIshiUnkFunc D_8095F7B0[] = { func_8095F210, func_8095F36C };
+static EnIshiMultiFunc sIshiGameplayKeepObjectDrawFuncs[] = { EnIshi_DrawGameplayKeepSmallRock,
+                                                              EnIshi_DrawGameplayKeepBoulder };
```

Comment:
```suggestion
static EnIshiMultiFunc sIshiGameplayKeepObjectDrawFuncs[] = { EnIshi_DrawGameplayKeepSmallRock,
                                                              EnIshi_DrawGameplayKeepBoulder, };
```
Same here. Think a trailing comma would foramt a bit better. (I'm generally not a big fan when clang formats multi lines all to the right)